backend/src/util/communityUser.ts

```diff
@@ -6,6 +6,8 @@ import { User } from '@model/User'
 
 const communityDbUser: dbUser = {
   id: -1,
+  gradidoID: '11111111-2222-4333-4444-55555555',
```

Comment:
Why this default value?

Reply:
there is no need here for an identical and unique gradidoID, it has only to satisfy that gradidoID is NOT NULL

Reply:
This one is fine